templates/apps/create.html

```diff
@@ -286,6 +293,26 @@ <h6> {{ vals.meta_title }}</h6>
         </div>
     </div>
 </div>
+{# In the custom app creation form, make the path a required field if a persistent volumet is selected #}
+{% if app.name == "Custom App" %}
+<script>
+    document.addEventListener('DOMContentLoaded', function () {
+        var pvcField = document.getElementById('Persistent Volume');
+        var pathField = document.getElementById('path');
+        var pathDiv = document.querySelector('.path_div');
+
+        pvcField.addEventListener('change', function () {
+            if (pvcField.value !== 'None') {
+                pathField.setAttribute('required', 'required');
+                pathDiv.classList.add('validate-item');
+            } else {
+                pathField.removeAttribute('required');
+                pathDiv.classList.remove('validate-item');
+            }
+        });
+    });
+</script>
+{% endif %}
```

Comment:
I'm okay with this, but just wanted to say, that it could also be done on server side with django